### DISCUSSION TITLE
Desktop does not start on the computer

### DISCUSSION THREAD

**Discussion by ghost538** [community/community#48923] [Other Feature Feedback, Questions, & Ideas]

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @ghost538 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
